lib/Platform/Logging.cpp

```diff
@@ -11,6 +11,7 @@
 #include <android/log.h>
 #elif defined(__APPLE__)
 #include <os/log.h>
+#include <stdio.h>
```

Comment:
This was the source of issues 😄 `vsnprintf` function wasn't automatically imported for visionOS

Reply:
from what I read in the docs, it's probably also a problem on macOS 11.3+ toolchains. weird they'd make such a fundamental change to their core libc headers after so many years 🤷

Reply:
@okwasniewski @matthargett it is the same for tvOS